examples/restify/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "restify-example",
   "private": true,
-  "version": "0.18.0",
+  "version": "0.15.0",
```

Comment:
Looking at the history of this file -- the first version was checked in as 0.18.0.

Reply:
Most likely core was 18 at that time and the original contributor was confused what our policy is?